.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,67 @@
+name: Bug Report
+description: Create a report with a procedure for reproducing the bug
+labels: "waiting-for-triage"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        > [!NOTE]
+        > Check [SECURITY policy](https://github.com/fluent/fluentd-docker-image/blob/master/SECURITY.md) first when you report a vulnerability issue.
```

Comment:
Thanks!

Do we need to add `SECURITY.md`?

Reply:
Here.

https://github.com/fluent/fluentd-docker-image/pull/395